### PR TITLE
{toolchain} [intel/2015.02] (REVIEW)

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by pescobar** [comment]
I have another PR ready for iomkl/2015.02 but first I need this one to be merged to devel branch

File: easybuild/easyconfigs/i/ifort/ifort-2015.2.164-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+versionsuffix = '-%s-%s' % (gcc, gccver)
+
+dependencies = [(gcc, gccver)]
+
+dontcreateinstalldir = 'True'
+
+# license file
+#import os
+#license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = '/import/bc2/soft/source/i/ifort/COM_L__FOR_RRB4-H33RVJNV.lic'
```
Comment (wpoely86) [inline easybuild/easyconfigs/i/ifort/ifort-2015.2.164-GCC-4.9.2.eb]: this is site specific I suppose? If so, remove it?

Comment (boegel) [inline easybuild/easyconfigs/i/ifort/ifort-2015.2.164-GCC-4.9.2.eb]: +1

File: easybuild/easyconfigs/i/icc/icc-2015.2.164-GCC-4.9.2.eb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+versionsuffix = '-%s-%s' % (gcc, gccver)
+
+dependencies = [(gcc, gccver)]
+
+dontcreateinstalldir = 'True'
+
+# license file
+#import os
+#license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = '/import/bc2/soft/source/i/icc/COM_L__CPP_RSCN-FRFDSRTL.lic'
```
Comment (boegel) [inline easybuild/easyconfigs/i/icc/icc-2015.2.164-GCC-4.9.2.eb]: switch back to the commented out line

we now know where you store your license file! mwoehahahaha! :P

**Comment by boegel** [comment]
looks fine, other than the `license_file` issue

Two requests:
i) please also include a HPL easyconfig using this new toolchain, by means of a test case
ii) please look into uploading a test report for this PR

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7fcf77972095111addd2 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 0 out of 8 (8 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/560db19070ecc2055047 for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**FAILED**
Build succeeded for 1 out of 8 (8 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8a6c4c0ab9b445d9dcf8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/10292d4f85a13a75ca4e for a full test report.

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a6e406f26c91310ce7ee for a full test report.

**Comment by boegel** [comment]
After a couple of hickups caused by issues with missing 'source' files and licensing troubles, all is well with the testing, and this can go in.

Thanks @pescobar!

**Pull request merged**